lib/scrabble.rb

```diff
@@ -1,18 +1,51 @@
 class Scrabble
 
+  def initialize
+    @array_of_word = []
+    @count = 0
+  end
+
   def score(word)
-    1
+    if word.length > 6
+      @count += 10
+    end
+    @array_of_word =[]
+    word.split('').each do |letter|
+      @array_of_word << letter.upcase
+    end
+      point_counter(@array_of_word)
+
+  end
+
+  def point_counter(array)
+    points =  array
+    points.map {|letters| SCRABBLE_POINTS.values_at(letters)}
```

Comment:
Remember to save the return result of .map to a variable since it doesn't change the original array. This is why you're getting an error on line 24 - you are inadvertently trying to add a string to an integer there.

Reply:
When setting the block parameter |letters|, remember to name it single instead of plural this variable is going to hold only one letter.

Reply:
The Hash method you've used .values_at will work for this task. However, you'll more commonly see it done with square bracket notation. For example: SCRABBLE_POINTS[letter] 
This strategy is both easier to read and a shorter to type.